index.html

```diff
@@ -85,95 +84,107 @@ <h3 class="article__title">Why do refugees need bikes?</h3>
                 Médecins Sans Frontières.
               </p>
-            </div>
+            </article>
           </div>
-
-          <div class="article">
+        </div>
+        
+          <article class="article">
             <img
               class="article__thumbnail"
-              src="bikes-for-refugees_logo.jpg"
-              alt=""
+              src="images/bikes-for-refugees_logo.jpg"
+              alt="bikes for refugees logo"
             />
-            <div class="article__content">
+            <article class="article__content">
               <h3 class="article__title">
-                <a class="article__title-link" href="#">Bike dropoff event in Edinburgh on May 11</a>
+                <a class="article__title-link" href="#"
+                  >Bike dropoff event in Edinburgh on May 11</a
+                >
               </h3>
-            </div>
-          </div>
-
-          <div class="article">
+            </article>
+          </article>
+        
+        
+          <article class="article">
             <img
               class="article__thumbnail"
-              src="edinburgh-damascus_thumbnail.png"
-              alt=""
+              src="images/edinburgh-damascus_thumbnail.png"
+              alt="help us cycle 4797km"
             />
-            <div class="article__content">
+            <article class="article__content">
               <h3 class="article__title">
-                <a class="article__title-link" href="#">Help us cycle 4,797km</a>
+                <a class="article__title-link" href="#"
+                  >Help us cycle 4,797km</a
+                >
               </h3>
               <p class="article__summary">
                 We need you to help us cycle 4797km, the distance of Edinburgh
                 to Damascus.
               </p>
-            </div>
-          </div>
-
-          <a
-            href="https://www.facebook.com/BikesforRefugeesScotland/events/?ref=page_internal"
-            target="_blank"
-            class="facebook-link"
-          >
-            <span class="fa fa-facebook-square"></span>
-            All Facebook events
-          </a>
-        </div>
-    </div>
+            </article>
+          </article>
+      
+
+        <a
+          href="https://www.facebook.com/BikesforRefugeesScotland/events/?ref=page_internal"
+          target="_blank"
+          class="facebook-link"
+        >
+          <span class="fa fa-facebook-square"></span>
+          All Facebook events
+        </a>
+        </sidebar>
+           </content>
 
-    <div class="footer">
+    <footer class="footer">
       <div class="footer__content">
         <p>
-          <strong>Bikes for Refugees Scotland is a volunteer-led project to help
-          people explore and make connections with their local communities.</strong>
+          <strong
+            >Bikes for Refugees Scotland is a volunteer-led project to help
```

Comment:
```suggestion
          <strong>
            Bikes for Refugees Scotland is a volunteer-led project to help
```
A small styling suggestion to put this closing tag `>` on the line above.